widgetry/src/assets.rs

```diff
@@ -1,3 +1,5 @@
+use crate::drawing::GfxCtx;
```

Comment:
Nit: Since these're exported from the crate, could just do these on the `use crate::{text, GeomBatch}` line